deps/chakrashim/src/jsrtisolateshim.cc

```diff
@@ -59,12 +63,6 @@ IsolateShim::~IsolateShim() {
 }
 
 /* static */ v8::Isolate * IsolateShim::New() {
-  // CHAKRA-TODO: Disable multiple isolate for now until it is fully implemented
```

Comment:
Because now we need atleast 2 isolate (one main isolate and another for debugger agent).